src/config_trapdoor.c

```diff
@@ -764,16 +765,45 @@ TbBool parse_trapdoor_door_blocks(char *buf, long len, const char *config_textna
                 COMMAND_TEXT(cmd_num),block_buf,config_textname);
           }
           break;
+      case 4: // SLABKIND
+          if (get_conf_parameter_single(buf, &pos, len, word_buf, sizeof(word_buf)) > 0)
+          {
+              k = get_id(slab_desc, word_buf);
+              doorst->slbkind[1] = k;
+              n++;
+          }
+          else
+          {
+              CONFWRNLOG("Incorrect slab name \"%s\" in [%s] block of %s file.",
+                  word_buf, block_buf, config_textname);
+              break;
+          }
+          if (get_conf_parameter_single(buf, &pos, len, word_buf, sizeof(word_buf)) > 0)
+          {
+              k = get_id(slab_desc, word_buf);
+              doorst->slbkind[0] = k;
+              n++;
+          }
+          else
+          {
+              CONFWRNLOG("Incorrect slab name \"%s\" in [%s] block of %s file.",
+                  word_buf, block_buf, config_textname);
+          }
+          if (n < 2)
+          {
+              CONFWRNLOG("Couldn't read \"%s\" parameter in [%s] block of %s file.",
+                  COMMAND_TEXT(cmd_num), block_buf, config_textname);
+              break;
+          }
+          break;
       case 5: // HEALTH
           if (get_conf_parameter_single(buf,&pos,len,word_buf,sizeof(word_buf)) > 0)
           {
             k = atoi(word_buf);
             if (i < DOOR_TYPES_COUNT)
```

Comment:
since you're moving it to DoorConfigStats, you no longer need the check on DOOR_TYPES_COUNT otherwisedoors would be limited to 5

Reply:
Changed it to gameadd.trapdoor_conf.door_types_count